src/bauhaus/bauhaus.c

```diff
@@ -1216,6 +1216,24 @@ gboolean dt_bauhaus_combobox_set_from_text(GtkWidget *widget, const char *text)
   return FALSE;
 }
 
+gboolean dt_bauhaus_combobox_set_from_value(GtkWidget *widget, int value)
```

Comment:
Just a comment, do you think it would be clearer to name this routine:
dt_bauhaus_combobox_set_from_index() for me value could be the text actually. Index make it clear that we are looking for the Nth value. What do you think?

Reply:
Hm... that's not what it does though? Setting from the number of the item is what the normal `dt_bauhaus_combobox_set` does. This finds the enum value (in case the combobox is setting an enum). So could be sparse and/or out-of-order. It could be called `dt_bauhaus_combobox_set_from_data`, but the fact that the enum value is internally stored in a field called _data_ to me feels like an implementation detail.

Reply:
I see, sorry I read wrongly the code :) All good then.